astropy/utils/iers/iers.py

```diff
@@ -43,10 +43,16 @@
            'LeapSeconds', 'IERS_LEAP_SECOND_FILE', 'IERS_LEAP_SECOND_URL',
            'IETF_LEAP_SECOND_URL']
 
+# maps specific download urls to the additional http headers needed for them -
+# populated where specific urls are given below.
+SPECIAL_HTTP_HEADERS_NEEDED = {}
```

Comment:
Is this even necessary? Can't you pass that in blindly regardless of URL? What if user change the value of `IERS_A_URL_MIRROR` at run time?

Reply:
My hypothesis was that other custom headers might be called for, but as you an @aarchiba pointed out maybe it's best to always do it...